spec/spec_helper_acceptance.rb

```diff
@@ -2,6 +2,9 @@
 require 'mustache'
 require 'open3'
 
+require 'spec_helper_integration' if ENV['USE_BEAKER']
```

Comment:
I wouldn't call this "spec_helper_integration" as that has connotations that it will be loaded directly by tests. I would set it as a regular file loaded from the support dir.

Also, if a user explicitly sets `USE_BEAKER` to false on the shell, this will be "false" (String) and still be "truthy"